Tests/SwiftDocCTests/Infrastructure/DocumentationContext/DocumentationContextTests.swift

```diff
@@ -510,10 +510,10 @@ class DocumentationContextTests: XCTestCase {
         let identifierPaths = context.knownIdentifiers.map { $0.path }.sorted(by: { lhs, rhs in lhs.count < rhs.count })
         XCTAssertEqual(identifierPaths, [
             // From the two file names
-            "/tutorials/Technology-file:-with---whitespace,-and_punctuation",
+            "/tutorials/Technology-file_-with---whitespace,-and_punctuation",
             // From the volume's title and the chapter's names, appended to their technology's identifier
-            "/tutorials/Technology-file:-with---whitespace,-and_punctuation/Volume_Section-Title:-with---various!-whitespace,-and/punctuation",
-            "/tutorials/Technology-file:-with---whitespace,-and_punctuation/Volume_Section-Title:-with---various!-whitespace,-and/punctuation/Chapter_Title:-with---various!-whitespace,-and/punctuation"
+            "/tutorials/Technology-file_-with---whitespace,-and_punctuation/Volume_Section-Title:-with---various!-whitespace,-and/punctuation",
+            "/tutorials/Technology-file_-with---whitespace,-and_punctuation/Volume_Section-Title:-with---various!-whitespace,-and/punctuation/Chapter_Title:-with---various!-whitespace,-and/punctuation"
```

Comment:
Is it expected that the `:` towards the end of this string ("Chapter_Title:") isn't replaced with `_`?

Reply:
No, that isn't expected, but how did the test pass 🤯.  `:` is a ADS (alternate data stream, think of it as macOS file forks) separator, the file would just be truncated to the `:` and the rest would be the name of the stream.